src/components/buttons/RegularButton.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const toplineR = css`
```

Comment:
What does the `R` mean? Better not use abbreviations in general, only if they are super obvious like HTTP.

If this variable is specifically for styles, I would also recommend to add the postfix `Styles` (like `toplineRStyles`)

Reply:
I just saw that you use this style only in one place. In this case, why not put it directly there?

Reply:
ToplineR is one of the Text styles defined in figma (and it is called there just like this, I also do not know what the `R` means).

I did not use this css in the styled component directly since @georgiee und @fischeversenker wanted to have a new typography.tsx with those text styles and until we have that one, I shall just do it like above.

Reply:
All right, if that's the name in Figma that is fine. Maybe you can ask Design if they can use more expressive names, but we don't need to wait for feedback here 👍